pkg/plugins/runtime/k8s/controllers/namespace_controller.go

```diff
@@ -29,11 +24,8 @@ type NamespaceReconciler struct {
 	kube_client.Client
 	Log logr.Logger
 
-	SystemNamespace         string
-	CNIEnabled              bool
-	SkipDefaultMeshCreation bool
-	ResourceManager         core_manager.ResourceManager
-	DefaultMeshTemplate     mesh_proto.Mesh
+	CNIEnabled      bool
+	ResourceManager core_manager.ResourceManager
```

Comment:
Is it used?

Reply:
I'll fix by the way it in the next PR that I'll be introducing